ext/oembed/ang/crmOembedSharing.ang.php

```diff
@@ -0,0 +1,23 @@
+<?php
+// Angular module oembedSharing.
+// @see https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_angularModules
+return [
+  'js' => [
+    'ang/crmOembedSharing.js',
+    'ang/crmOembedSharing/*.js',
+    'ang/crmOembedSharing/*/*.js',
+  ],
+  'css' => [
+    'ang/crmOembedSharing.css',
+  ],
+  'partials' => [
+    'ang/crmOembedSharing',
+  ],
+  'requires' => ['crmUi', 'crmUtil'],
+  'settings' => [],
+  'basePages' => [],
+  'bundles' => ['bootstrap3'],
+  'exports' => [
+    'oembed-sharing' => 'A',
```

Comment:
Just curious what 'A' means/stands for. Is it that the oEmbed is done using Angular?

Reply:
(Explained the other comment)